amm/interp/src/main/scala/ammonite/runtime/Interpreter.scala

```diff
@@ -359,6 +359,7 @@ class Interpreter(val printer: Printer,
       tag
     ) match {
       case None =>
+        printer.out("Compiling " + wrapperName.backticked + "\n")
```

Comment:
Let's pull the name of the file out of the `source` and use that, including the `.scala`.